extensions/http-curl/client/HTTPClient.cpp

```diff
@@ -265,36 +324,6 @@ bool HTTPClient::submit() {
   }
 
   logger_->log_debug("Finished with %s", url_);
-  std::string key = "";
```

Comment:
This logic was partial, standard non-compliant and buggy. (The concrete issue was that it included extra \r\n's in the last header value, but there are more problems with it). See the reimplementation below.

Reply:
The more I think about it, I created https://issues.apache.org/jira/browse/MINIFICPP-981. Would love to hear your thoughts on that. curl bugs because we moved fast and loose with a limited number of devs means we're going to have more bugs. Even if people come around and locate them we still have a lot of risk because CURL has a large and constantly evolving API. curlpp has dedicated their project to an http client and I'd love to stop the maintenance of that and instead only wrap it.